dace/transformation/dataflow/redundant_array.py

```diff
@@ -2,13 +2,84 @@
 """ Contains classes that implement a redundant array removal transformation.
 """
 
+import dace
```

Comment:
(style) please import the innermost module (`data as dt`, seems like?) instead of importing the entire package

Reply:
In one of the (many) earlier versions, I needed `dace.Memlet`. Is that top-level or not?

Reply:
`from dace import memlet` (`as mmlt` or `as mm`) also works.